src/storage-controller/src/lib.rs

```diff
@@ -474,14 +475,28 @@ where
             location,
             grpc_client: self.config.parameters.grpc_client.clone(),
         };
+        for id in instance.active_ingestions() {
+            self.collections
+                .get_mut(id)
+                .expect("instance contains unknown ingestion")
```

Comment:
Can we make this a soft panic instead? I'm wary of panics that could bring down envd, if it's not immediately obvious that we can't hit them. And I don't think that's obvious here.